CHANGELOG.md

```diff
@@ -0,0 +1,98 @@
+# Changelog
+
+## [Unreleased]
+
+## [v0.1.5] - 2021-01-05
+
+### Added
+
+- Allow support for semicolumn and tabulation in CSV files
+- Collect current distance from OR-Tools
+- Allow initial routes when partitions are required. Note that, this feature is available for preprocessing/partitions but is under development for split based algorithms (`split_solve` and `dichotomous`)
+- Position field for activities
+- Sheduling and initial routes are now compatible
+- Unfeasible missions are removed from routes [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- Log reasons of unfeasible missions [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Read and log OR-Tools version
+- Route costs [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+- Log of matrix computation [#52](https://github.com/Mapotempo/optimizer-api/pull/52)
+- [WIP] Allow multidepot within clustering [#56](https://github.com/Mapotempo/optimizer-api/pull/56)
```

Comment:
Since it is [WIP] I would add limitations if it's simple to explain

Reply:
@senhalil Could you provide some details on the current limitations ?

Reply:
At the moment, the depot locations only influence the initialization of the clusters and the clusters can convergence to regions away from their depots under certain conditions (density of the points, distribution of the depots).